packages/buefy-next/src/components/menu/MenuItem.vue

```diff
@@ -22,7 +22,7 @@
                 v-else
                 name="label"
                 :expanded="newExpanded"
-                :active="newActive"
+                :modelValue="newActive"
```

Comment:
I think we should leave it as "active", because this is given as a [slot prop for scoped slots](https://vuejs.org/guide/components/slots.html#scoped-slots). The example of the `Menu` component shows an example use of the `expanded` slot prop:
https://github.com/ntohq/buefy-next/blob/ca1c73838c2266370577fd2b4fd4139da4a6f19e/packages/docs/src/pages/components/menu/examples/ExSimple.vue#L6-L9

You can see how the `active` slot prop works if you replace the above code with the following:
```vue
        <template #label="props">
          Administrator ({{ props.active ? 'active' : 'inactive' }})
          <b-icon class="is-pulled-right" :icon="props.expanded ? 'menu-up' : 'menu-down'"></b-icon>
        </template>
```